test/unit/fact_parser_test.rb

```diff
@@ -60,18 +60,14 @@ class FactParserTest < ActiveSupport::TestCase
     end
   end
 
-  test "#normalize_interfaces converts custom-case interface names to be downcase" do
-    assert_equal ['eth0', 'eth0.0', 'em1'], parser.send(:normalize_interfaces, ['ETH0', 'Eth0.0', 'eM1'])
-  end
```

Comment:
We don't want this, why? After you explain, please reverse the semantics of this test rather than picking random string below (`Local_Area_Connection_2`).

Reply:
method `normalize interfaces`has been removed from `FactParser`. See conversation above  about downcase in `PuppetFactParser`.

Reply:
Thanks for explanation, just fix this behavior in an extra test so we won't delete the line in the future please.

Marked all as resolved, we are all set!

Reply:
tx!, behavior is covered by existing tests:
* downcase behavior for facter < v3.0 is covered by `#interfaces are mapped case-insensitively and parses Windows LAN name when facter < v3.0`.
* preserve case behavior for other facters is covered by `#interfaces names are not mangled when facter >= v3.0` for `PuppetFactParser` and `#interfaces gets facts hash for desired interfaces, keeping same values it gets from parser` for `FactParser`.

Reply:
Oh ok then, I am good.